src/main/java/lotto/service/LottoService.java

```diff
@@ -5,47 +5,24 @@
 import lotto.view.InputView;
 import lotto.view.ResultView;
 
-import java.util.HashMap;
-import java.util.LinkedList;
-import java.util.List;
 import java.util.Map;
 
 public class LottoService {
     public static void play() {
-        int boughtMoney = InputView.inputBoughtMoney();
-        validateBoughtMoney(boughtMoney);
-        int lottoCount = LottoServiceCalculator.getLottoCount(boughtMoney);
-        ResultView.printLottoCount(lottoCount);
-        List<Lotto> lottoList = createLottoList(lottoCount);
-        ResultView.printLottoNumbers(lottoList);
+        int boughtMoney = new BoughtMoney(InputView.inputBoughtMoney()).getBoughtMoney();
+        int totalCount = LottoServiceCalculator.getLottoCount(boughtMoney);
+        ManualLottos manualLottos = new ManualLottos(new ManualLottosCount(totalCount, InputView.inputManualLottoCount()));
+        int autoLottoCount = totalCount - manualLottos.getManualLottosCount();
+        manualLottos.createManualLottos(InputView.inputManualLottoNumbers(manualLottos.getManualLottosCount()));
```

Comment:
`ManualLottosCount`에게 조금 더 큰 역할을 맡겨봐도 될 것 같아요.
만약 `LottosCount`라 이름을 바꿔 인스턴스 변수로 `manualLottoCount`와 `autoLottoCount`를 둘 다 갖고 있을 수 있다면
아래와 같은 코드도 가능할 것 같습니다.

```suggestion
        LottosCount lottosCount = new LottosCount(boughtMoney, InputView.inputManualLottoCount());
        ManualLottos manualLottos = new ManualLottos(lottosCount);
        manualLottos.createManualLottos(InputView.inputManualLottoNumbers(lottosCount));
```

아, 추가적으로 위 코드 제안은 `BoughtMoney`가 `LottoServiceCalculator.getLottoCount`의 기능까지 수행한다는 전제 하의 코드입니다. 관련해서 `BoughtMoney`에도 코멘트를 남겨두었으니 참고해보세요!